scripts/build.py

```diff
@@ -221,7 +233,8 @@ def generate_app_yaml(deploy_mode=False, maintenance_mode=False):
         content = content.replace('version: default', '')
         # The FIREBASE_AUTH_EMULATOR_HOST environment variable is only needed to
         # test locally, and MUST NOT be included in the deployed file.
-        content = re.sub('  FIREBASE_AUTH_EMULATOR_HOST: ".*"\n', '', content)
+        for env_variable in ENV_VARS_TO_REMOVE_FROM_DEPLOYED_APP_YAML:
+            content = re.sub('  %s: ".*"\n' % env_variable, '', content)
```

Comment:
I wonder if it's worth verifying that the string actually exists before deleting it. If it doesn't exist in the "pre" state then that might indicate a dev error that should be caught. It would also prevent the code here from silently going out of date.

Reply:
Makes sense. Done.